options/posix/include/bits/posix/posix_signal.h

```diff
@@ -91,6 +91,8 @@ int sigtimedwait(const sigset_t *__restrict set, siginfo_t *__restrict info, con
 int sigwait(const sigset_t *__restrict set, int *__restrict sig);
 int sigwaitinfo(const sigset_t *__restrict set, siginfo_t *__restrict info);
 
+int sigisemptyset(const sigset_t *set);
```

Comment:
This is a GNU extension, maybe wrap in a check for the glibc option?

Reply:
Good catch, wrapped as requested.